frontend/src/queries/nodes/DataTable/ColumnConfigurator/ColumnConfigurator.tsx

```diff
@@ -0,0 +1,233 @@
+import './ColumnConfigurator.scss'
+import { BindLogic, useActions, useValues } from 'kea'
+import { LemonButton } from 'lib/components/LemonButton'
+import { dataTableLogic } from '~/queries/nodes/DataTable/dataTableLogic'
+import { IconTuning, SortableDragIcon } from 'lib/components/icons'
+import { RestrictedArea, RestrictedComponentProps, RestrictionScope } from 'lib/components/RestrictedArea'
+import { LemonCheckbox } from 'lib/components/LemonCheckbox'
+import clsx from 'clsx'
+import { PropertyKeyInfo } from 'lib/components/PropertyKeyInfo'
+import { Tooltip } from 'lib/components/Tooltip'
+import { CloseOutlined, LockOutlined } from '@ant-design/icons'
```

Comment:
We'd have `IconClose` and `IconClose` to squish those ants with lemons :)

Reply:
Indeed! Actually the Lock icon wasn't even needed anymore. The previous `ColumnConfigurator` that I copied over (under `lib/components/ResizableTable/index.tsx` _ugh_), locked some columns. I didn't see a need for that, so removed that part totally... and replaced the Close icon with the ant free version.